src/jwt/strategy.js

```diff
@@ -17,16 +18,26 @@ export default () => {
   }
 
   return new Strategy(options,
-    (JWTPayload, next) => {
-      // usually this would be a database call:
-      // var user = users[_.findIndex(users, {id: JWTPayload.id})]
-      // TODO: fix https://github.com/Human-Connection/Nitro-Backend/issues/41
-      /* eslint-disable */
-      if (true) {
-      /* eslint-enable */
-        next(null, {})
+    async (JWTPayload, next) => {
+      const session = driver.session()
+      const result = await session.run(
+        'MATCH (user:User {id: $userId}) ' +
+        'RETURN user {.id, .slug, .name, .avatar, .email, .password, .role} as user LIMIT 1',
+        {
+          userId: JWTPayload.id
+        }
+      )
+      session.close()
+      const [currentUser] = await result.records.map((record) => {
+        return record.get('user')
+      })
+
+      if (currentUser) {
+        delete currentUser.password
```

Comment:
you don`t need to delete the password here when you do not return it in the above query

Reply:
sorry, new to neo4j. I'll edit that right now!